nautobot/circuits/views.py

```diff
@@ -135,6 +135,8 @@ class CircuitUIViewSet(NautobotUIViewSet):
     queryset = Circuit.objects.all()
     serializer_class = serializers.CircuitSerializer
     table_class = tables.CircuitTable
+    # NOTE: This is how `NautobotUIViewSet` would define use_new_ui attr
+    # use_new_ui = ["list", "retrieve"]
```

Comment:
Why only here? We're using NautobotUIViewSet in a bunch of views:

```
nautobot/dcim/views.py:2815:class DeviceRedundancyGroupUIViewSet(NautobotUIViewSet):
nautobot/dcim/views.py:2839:class InterfaceRedundancyGroupUIViewSet(NautobotUIViewSet):
nautobot/circuits/views.py:97:class ProviderUIViewSet(NautobotUIViewSet):
nautobot/circuits/views.py:128:class CircuitUIViewSet(NautobotUIViewSet):
nautobot/circuits/views.py:181:class ProviderNetworkUIViewSet(NautobotUIViewSet):
nautobot/extras/views.py:1518:class JobButtonUIViewSet(NautobotUIViewSet):
nautobot/extras/views.py:1777:class RoleUIViewSet(viewsets.NautobotUIViewSet):
```

Reply:
It's random, and it's simply a note to users/anyone planning on getting NautobotUIViewSet new_ui ready on how to implement it.